quetz/main.py

```diff
@@ -1052,7 +1024,12 @@ def handle_package_files(
         if force:
             auth.assert_overwrite_package_version(channel_name, package_name)
 
-        logger.debug(f"adding file '{file.filename}' to channel '{channel_name}'")
+        file.file._file.seek(0, os.SEEK_END)
+        size = file.file._file.tell()
+        total_size += size
```

Comment:
for the channel size limit: do we want to check whether all the files can be uploaded or we do that one by one and fail after N files have been already uploaded

Reply:
I like to fail early instead of halfway through.